pages/blog/[slug].tsx

```diff
@@ -51,17 +52,11 @@ function BlogTemplate({ file, siteConfig, prevPage, nextPage }) {
             title: frontmatter.title,
             description: excerpt,
             images: [
-              {
-                url:
-                  'https://res.cloudinary.com/forestry-demo/image/upload/l_text:tuner-regular.ttf_70:' +
-                  encodeURIComponent(frontmatter.title) +
-                  ',g_north_west,x_270,y_95,w_840,c_fit,co_rgb:EC4815/l_text:tuner-regular.ttf_35:' +
-                  encodeURIComponent(frontmatter.author) +
-                  ',g_north_west,x_270,y_500,w_840,c_fit,co_rgb:241748/v1581087220/TinaCMS/tinacms-social-empty.png',
-                width: 1200,
-                height: 628,
-                alt: frontmatter.title + ` | TinaCMS Blog`,
-              },
+              openGraphImage(
+                frontmatter.title,
+                '| TinaCMS Blog',
```

Comment:
Space is needed as on https://tinacms-site-next-git-refactor-open-graph-images.tinacms.vercel.app/blog/inline-editing-project it displays `Inline Editing Revamped| TinaCMS Blog` 

```suggestion
                ' | TinaCMS Blog',
```